vectordatasource/transform.py

```diff
@@ -8912,7 +8912,7 @@ def tags_set_ne_min_max_zoom(ctx):
             # way, we probably don't want to see them at zoom, which is lower
             # than most of the curated NE min zooms. see issue #1826 for more
             # information.
-            props['min_zoom'] = max(6, props['min_zoom'])
```

Comment:
Please make this 8 instead of 6.

And set the **kind** to `disputed_country`.

Reply:
This will require more matching against `map_units` from Natural Earth, not `countries` table, so please also include that join change (keeping **wikidataid** as the join column) in this PR.

That table also has _more_ "countries" than we possibly want... so consider a related PR to strip out some of those (there are up to 70).

Reply:
EG **Somaliland** and **Northern Cyprus**.

Reply:
Looks like this will need to use the source DBF that makes the map_units (which includes countries) as some countries aren't included in the map units file alone?

Reply:
Nagorno-Karabakh also shows up at min_zoom 1 now, because it starts as "kind" = `unrecognized`, so the if test wrapping this doesn't trigger, so the min_zoom isn't updated as part of the join.

Reply:
As added protection, in https://github.com/tilezen/vector-datasource/blob/master/yaml/places.yaml#L141-L152 we should also change the min_zoom to a case:

```
 - filter: {name: true, place: [country,unrecognized]}
    # note: min_zoom needs to be smaller than any min_label in
    # ne_10m_admin_0_countries to make sure we're not dropping any by not being
    # included in the PostgreSQL index.
    #
    # note: also, we knock this down to 6 during post-processing for any country
    # which doesn't match an NE curated one.
    min_zoom: 
      case:
        - when: { place: country }
          then: 1
        - else: 8
    output:
      <<: [*output_properties, *ne_zooms]
      kind: {col: place}
    table: osm
```

Where the else can be 8 or some other value.

Reply:
Only this comment remains unaddressed:

> Nagorno-Karabakh also shows up at min_zoom 1 now, because it starts as "kind" = unrecognized, so the if test wrapping this doesn't trigger, so the min_zoom isn't updated as part of the join.

Fix would be something like:

> elif props.get('kind') in ('country', 'unrecognized'):